app/controllers/patients_controller.rb

```diff
@@ -119,7 +116,7 @@ def find_patient
                   :procedure_cost, :check_number, :date_of_check]
   ].freeze
 
-  OTHER_PARAMS = [:urgent_flag, :initial_call_date, :pledge_sent].freeze
+  OTHER_PARAMS = [:urgent_flag, :initial_call_date, :pledge_sent, :pledge_sent_by, :pledge_sent_at, :last_edited_by].freeze
```

Comment:
hmm, I don't think you need the `pledge_sent_by`/`pledge_sent_at` here -- the `patient_params` deals explicitly with the payload sent to the controller for mass assignment (basically saying 'it's okay to change all of these fields at once, but only these fields'), and since I don't think we're setting them from the form, we can probably skip changing this set. Does that sound right to you?

Reply:
Yes, I think you're right!